src/isar_turtlebot/turtlebot/turtlebot.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+from isar_turtlebot.models.turtlebot_status import Status
+from isar_turtlebot.ros_bridge import RosBridge
+from isar_turtlebot.turtlebot.taskhandlers.driveto import DriveToHandler
+from isar_turtlebot.turtlebot.taskhandlers.takeimage import TakeImageHandler
+from isar_turtlebot.turtlebot.taskhandlers.takethermalimage import (
+    TakeThermalImageHandler,
+)
+from isar_turtlebot.turtlebot.taskhandlers.taskhandler import TaskHandler
+from isar_turtlebot.utilities.inspection_pose import get_inspection_pose
+from isar_turtlebot.utilities.pose_message import decode_pose_message
+from robot_interface.models.geometry.pose import Pose
+from robot_interface.models.mission.task import (
+    DriveToPose,
+    TakeImage,
+    TakeThermalImage,
+    Task,
+)
+
+
+class Turtlebot:
+    """Task manager for Turtlebot."""
+
+    def __init__(
+        self,
+    ) -> None:
+
+        self.bridge: RosBridge = RosBridge()
+        self.status: Optional[Status] = None
+
```

Comment:
```suggestion
    def __init__(
        self,
        bridge: RosBridge
    ) -> None:

        self.bridge: RosBridge = bridge
        self.status: Optional[Status] = None
```
This way the `RosBridge` may be passed to the class instead such that it's easier to create a `MockRosBridge` to replace the bridge for testing. (This is dependency injection in practice, you don't need a package like `injector` really)